src/utils/url.ts

```diff
@@ -105,6 +105,22 @@ export const isValidUrl = (url: string): boolean => {
   }
 };
 
+// @debt possibly merge this and isValidUrl right above
```

Comment:
not sure if it was moved from some other place but if this is a new function then why a @debt comment from the start?

Reply:
removed d0f3e28